src/python/aqueduct_executor/operators/connectors/tabular/extract.py

```diff
@@ -29,7 +29,10 @@ def replace_today() -> str:
 class RelationalParams(models.BaseParams):
     # The query cannot be used until `apply_placeholders()` is called on it. This flushes out
     # any user-defined tags like `{{today}}`.
-    query_is_usable: bool = False
+    # _query_is_usable is not passed in via the spec, so we prefix it with an _ to ensure that
```

Comment:
@kenxu95 Could you confirm that my understanding is correct here. I believe that this instance variable is just used for tracking state, but it isn't passed via the job spec ever from Go to Python?

Reply:
yeah this never created from Golang, its just lifecycle management so that we can't use RelationalParams ever without expanding parameter values.